modules/javafx.graphics/src/main/native-glass/gtk/fp_pipewire.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#ifdef HEADLESS
```

Comment:
I don't have a strong opinion. This file is unchanged relative to the original in AWT. The other three already have some changes, so I'd probably lean toward removing it.

Reply:
With the headless platform that I started in the sandbox project, there is the concept of headless, but rather than compiling native code with `#ifdef HEADLESS` the current approach is to avoid those native compilations at all.
However, I don't exclude that some parts of a file might make sense for a headless glass platform, where other parts should not be considered. In that case, the `ifdef HEADLESS` and in general the compile option `-DHEADLESS` would make sense.
While this is not needed at this moment (and not even sure it will ever be needed), I am slightly in favor of keeping it here.

Reply:
I already did the removal earlier, but I can bring it back once we come to an agreement.

Reply:
Johan makes a good point, so go ahead and add it back.

Reply:
Worth noting is that this `HEADLESS` check is just an assertion to verify that the build logic properly excludes the file from the build when building headless. It otherwise does nothing.

Reply:
The `HEADLESS` check is back.